lib/content-services/src/lib/search/components/search-number-range/search-number-range.component.ts

```diff
@@ -84,13 +85,26 @@ export class SearchNumberRangeComponent implements SearchWidget, OnInit {
 
         this.enableChangeUpdate = this.settings?.allowUpdateOnChange ?? true;
         this.updateDisplayValue();
+        this.context.populateFilters
+            .asObservable()
+            .pipe(first())
+            .subscribe((filtersQueries) => {
+                if (filtersQueries[this.id]) {
```

Comment:
same here like in previous files - please store filtersQueries[this.id] in const or use rjxs operator to get only that single value to avoid repeating